schemas/codex/packageCollection.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "Collection of packages",
+  "properties": {
+    "packages": {
+      "type": "array",
+      "description": "Collection of packages",
+      "packages": {
+        "type": "object",
+        "$ref": "package.json"
+      }
+    },
+    "resultInfo": {
```

Comment:
seems that this section should look like 
`"resultInfo": {
  "description": "Result information schema",
  "$ref": "../resultInfo.schema"
    }`
this should fix jenkins  error `ERROR: lint-raml-cop: resultInfo.schema: Missing "description" for: value`

Reply:
Thanks @natali-zaitseva for talking a look and these details -- resolved these!